### ISSUE TITLE
BCommits Ctrl+D suddenly doesn't work properly

### ISSUE THREAD

**Issue by Archdsenna** [junegunn/fzf.vim#1556]
### Checklist

- [X] I have fzf 0.54.0 or later
- [X] I have searched through the existing issues

### Output of `:echo system(fzf#exec() .. ' --version')`

0.54.3 (af4917d)

### OS

- [ ] Linux
- [X] macOS
- [ ] Windows
- [ ] Etc.

### Problem / Steps to reproduce

Today, when I was using fzf.vim's ctrl+D to view the changes in the commit, I suddenly found that ctrl+D did not work properly. I pressed ctrl+D, but there was no result. But two weeks ago, it was working properly. I checked all the mappings in the system, including vim, and there was no key binding that directly conflicted with ctrl+D. I don't understand what went wrong.
![image](https://github.com/user-attachments/assets/beac1477-a6fc-4157-b935-e03c17a87045)

**Comment by Archdsenna** [comment]
The strange thing is that ctrl+S and ctrl+Y work fine.

**Comment by Archdsenna** [comment]
I bound ctrl+D to other shortcuts, but it didn't work properly either, including vim/nvim. I don't think this is a problem with fzf.vim. Does anyone have any suggestions or ideas? Thanks in advance

**Comment by Archdsenna** [comment]
Sorry, after I checked the shortcut key settings of Mac again, I found that ctrl+D was bound to display desktop, which caused ctrl+D to be hijacked by Mac. Everything is fine now.
​
![img_v3_02cp_607a24d6-cf51-4ebd-910f-fe87d5743ddg](https://github.com/user-attachments/assets/e9176101-4b00-40b0-aa7b-9c355c29db54)

Here I would like to thank everyone who has contributed to fzf.vim. I will close this issue
